ambuda/models/user_permissions.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
```

Comment:
Since this is short, you can keep this in `models.auth`.

Reply:
Ah, there's no model.auth -- but there should be! Let's rename `models.users` and combine it with this.